docs/index.rst

```diff
@@ -2,63 +2,89 @@
 sunpy core Documentation
 ************************
 
+.. image:: tour.svg
+  :align: center
+
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
 It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
```

Comment:
I feel like this sentence is missing a "and a bunch of other stuff" line lol

Reply:
I think you commented on the wrong line and I'm not quite sure what you mean?